### ISSUE TITLE
"Install the Azure CLI on Linux" should focus on that and tell URL

### ISSUE THREAD

**Issue by sdettmer** [MicrosoftDocs/azure-docs-cli#3183]
### Prerequisites

- [X] Write a descriptive title.
- [X] Search the existing issues.
- [X] I am reporting the documentation problem for the current version of Azure CLI.

### Azure CLI version being used

bash: az: command not found

### Link to affected document

https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt

### Description of the documentation error

The article "Install the Azure CLI on Linux" does not mention the URL where to download packages in case "you may manually install the Azure CLI on Linux by selecting the Install script option" (I like the "az" script (or binary?) as TGZ). This makes it very time consuming: instead of "Browse downloads [here]" and a two minutes job, it is a "25 minutes to read" (!!!) posting which forgets to mention the URL where to download a tarbal from.

### Suggested fix

The article should focus on installing and for example mention the URL where to download the "az" script (or binary?) in case "you may manually install the Azure CLI on Linux by selecting the Install script option", but instead it repeats APT basics like proxy configuration, where a link would be suited better. This would save readers time and if reading time would be 2 minutes instead 25 minutes, it would be ten times more efficient!

**Comment by AjayKumar-MSFT** [comment]
Adding doc metadata info: 



---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7895b9a-9a6a-cd98-6947-0e4a26eda9a3
* Version Independent ID: a3f27295-3e2e-a36d-02c2-8897295fc443
* Content: [Install the Azure CLI on Linux](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt)
* Content Source: [docs-ref-conceptual/install-azure-cli-linux.md](https://github.com/MicrosoftDocs/azure-docs-cli/blob/main/docs-ref-conceptual/install-azure-cli-linux.md)
* Service: **azure-cli**
* GitHub Login: @jiasli
* Microsoft Alias: **jiasli**

**Comment by JamesTran-MSFT** [comment]
@sdettmer 
Thanks for your feedback! I've assigned this issue to the author who will investigate and update as appropriate.

**Comment by jiasli** [comment]
**Install script** means this page: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=script

It actually installs from PyPI (https://pypi.org/) with `pip` so the URL doesn't necessarily need to be documented. Actually, that document hasn't been updated for some time. We do have a plan to provide a new `pip` installation guidance: https://github.com/Azure/azure-cli/issues/20476

**Comment by sdettmer** [comment]
Thank you for clarifying.
In meantime I noticed that this "script", which I needed to perform
essentially a HTTP POST to upload a file, is 70 Megabytes huge. There is an
installer script that appears to ask the user for consent for each action
against the system, but in fact it has hard-coded "consent everything"
built-in (funny detail: a comment in the installer says it would be
hardcoded because of some "review comments"). The "script" installer
contains many many files and even includes binaries, possibly a python
runtime, who knows. I'm afraid it is beyond repair, so providing a direct
download URL to this ugly installer of a 70 MB "script" indeed seems not
needed. So whatever it is, it is not a script.
(I'm afraid the installed I used does not installs via PIP but it extracts
a complete run time including python binaries, but possibly there are so
many versions and variants flying around that surely it's hard to keep
track of them).
